iosHackthonApp/Modules/Features/AddMeal/Sources/AddMeal/AddMealView.swift

```diff
@@ -102,19 +102,21 @@ private extension AddMealView {
         @Binding var info: String
         @Binding var quantity: Int
 
+        let textFieldStyle = GreyTextFieldStyle()
```

Comment:
There isn't a viewModel for this file as it was proving difficult with the textFields and the views being separate components. It will take some work to figure it out so I will tackle that separately